statsmodels/genmod/tests/test_glm.py

```diff
@@ -1318,16 +1365,190 @@ def test_wtd_patsy_missing():
+                           exog=self.data.exog[['INCOME', 'SOUTH']],
+                           family=family2).fit()
+
+
+class TestTweedieSpecialLog1(CheckTweedieSpecial):
+    def __init__(self):
+        super(TestTweedieSpecialLog1, self).__init__()
+        family1 = sm.families.Poisson(link=sm.families.links.log)
+        self.res1 = sm.GLM(endog=self.data.endog,
+                           exog=self.data.exog[['INCOME', 'SOUTH']],
+                           family=family1).fit()
+        family2 = sm.families.Tweedie(var_power=1, link_power=0)
+        self.res2 = sm.GLM(endog=self.data.endog,
+                           exog=self.data.exog[['INCOME', 'SOUTH']],
+                           family=family2).fit()
+
+
+class TestTweedieSpecialLog2(CheckTweedieSpecial):
+    def __init__(self):
+        super(TestTweedieSpecialLog2, self).__init__()
+        family1 = sm.families.Gamma(link=sm.families.links.log)
+        self.res1 = sm.GLM(endog=self.data.endog,
+                           exog=self.data.exog[['INCOME', 'SOUTH']],
+                           family=family1).fit()
+        family2 = sm.families.Tweedie(var_power=2, link_power=0)
+        self.res2 = sm.GLM(endog=self.data.endog,
+                           exog=self.data.exog[['INCOME', 'SOUTH']],
+                           family=family2).fit()
+
+
+class TestTweedieSpecialLog3(CheckTweedieSpecial):
+    def __init__(self):
+        super(TestTweedieSpecialLog3, self).__init__()
+        family1 = sm.families.InverseGaussian(link=sm.families.links.log)
+        self.res1 = sm.GLM(endog=self.data.endog,
+                           exog=self.data.exog[['INCOME', 'SOUTH']],
+                           family=family1).fit()
+        family2 = sm.families.Tweedie(var_power=3, link_power=0)
+        self.res2 = sm.GLM(endog=self.data.endog,
+                           exog=self.data.exog[['INCOME', 'SOUTH']],
+                           family=family2).fit()
+
+
+def testTweediePowerEstimate():
+    data = sm.datasets.cpunish.load_pandas()
+    y = [1.00113835e+05,   6.89668315e+03,   6.15726842e+03,
+         1.41718806e+03,   5.11776456e+02,   2.55369154e+02,
+         1.07147443e+01,   3.56874698e+00,   4.06797842e-02,
+         7.06996731e-05,   2.10165106e-07,   4.34276938e-08,
+         1.56354040e-09,   0.00000000e+00,   0.00000000e+00,
+         0.00000000e+00,   0.00000000e+00]
+    model1 = sm.GLM(y, data.exog[['INCOME', 'SOUTH']],
+                    family=sm.families.Tweedie(var_power=1.5, link_power=0))
+    res1 = model1.fit()
+    model2 = sm.GLM((y - res1.mu) ** 2,
+                    np.column_stack((np.ones(len(res1.mu)), np.log(res1.mu))),
+                    family=sm.families.Gamma(sm.families.links.log))
+    res2 = model2.fit()
+    # Sample may be too small for this...
+    # assert_allclose(res1.scale, np.exp(res2.params[0]), rtol=0.25)
+    p = model1.tweedie_estimate_power(res1.mu)
+    assert_allclose(p, res2.params[1], rtol=0.25)
```

Comment:
do you have the estimate for the power in R?   just as reference if it's very different